+siibra/+core/Parcellation.m

```diff
@@ -1,32 +1,32 @@
 classdef Parcellation < handle
     properties
-        Id
-        Name
-        Atlas
-        Modality
-        Description
-        Graph
-        Spaces
+        id
+        name
+        atlas
+        modality
+        desciption
+        regionTree
+        spaces
     end
 
     methods
         function parcellation = Parcellation(parcellation_json, atlas)
-            parcellation.Id = strcat(parcellation_json.id.kg.kgSchema, '/', parcellation_json.id.kg.kgId);
-            parcellation.Name = parcellation_json.name;
-            parcellation.Atlas = atlas;
-            parcellation.Modality = parcellation_json.modality;
+            parcellation.id = strcat(parcellation_json.id.kg.kgSchema, '/', parcellation_json.id.kg.kgId);
+            parcellation.name = parcellation_json.name;
+            parcellation.atlas = atlas;
+            parcellation.modality = parcellation_json.modality;
             if ~ isempty(parcellation_json.infos)
-                parcellation.Description = parcellation_json.infos(1).description;
+                parcellation.desciption = parcellation_json.infos(1).description;
             end
 
             % link spaces from atlas
-            parcellation.Spaces = Space.empty;
+            parcellation.spaces = siibra.core.Space.empty;
             % retrieve available spaces from atlas
             for available_space_index = 1:numel(parcellation_json.availableSpaces)
```

Comment:
index variable names is one place where brevity is often favored in MATLAB style, e.g. just `idx`